packages/apollo-language-server/src/engine/index.ts

```diff
@@ -85,8 +85,12 @@ export class ApolloEngineClient extends GraphQLDataSource {
     return this.execute<ListServices>({
       query: LIST_SERVICES,
       variables
-    }).then(({ data, errors }) => {
+    }).then(({ data, errors, extensions }) => {
       // use error logger
+      if (extensions && extensions.nonWebAuthErrorMessage) {
```

Comment:
can we not use `?.` in this codebase though?

Reply:
Unfortunately not, it's not supported I think 🙁